app/components/channel_intro/index.js

```diff
@@ -16,12 +16,12 @@ function mapStateToProps(state) {
     const currentUser = getCurrentUser(state);
 
     let currentChannelMembers = [];
-    if (currentChannel.type === Constants.DM_CHANNEL) {
+    if (currentChannel.type === General.DM_CHANNEL) {
         const otherChannelMember = currentChannel.name.split('__').find((m) => m.id !== currentUser.id);
```

Comment:
I fixed this in my PR but I figure we can just add it in here. Can you change the `m.id` to just `m`? That'll fix a small bug where the wrong `otherChannelMember` is chosen.